src/Mhwd.cpp

```diff
@@ -564,21 +565,25 @@ MHWD::STATUS Mhwd::uninstallConfig(Config *config)
     }
     else
     {
-        // TODO: Should we check for local requirements here?
-
-        // Run script
-        if (!runScript(installedConfig, MHWD::TRANSACTIONTYPE::REMOVE))
+        // Should we check for local requirements here?
```

Comment:
That comment should be removed long time ago. You can see that it only confused you (that's why comments should be the last resort and should not be widely used).